docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -16,15 +16,24 @@ cluster you want to register. You can use `kubectl config get-contexts` to
 verify that the correct cluster is selected, or `kubectl config use-context
 ${CONTEXT_NAME?}` to switch to cluster `CONTEXT_NAME`.
 
-Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use.
+Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use. This will create:
+- The `teleport` namespace
+- The `teleport-sa` service account
+- The `teleport-role` cluster role
+- The `teleport-crb` cluster role binding
 
 You can connect multiple Kubernetes clusters to Teleport from one `kubeconfig` if it contains multiple entries. Use
 [merge-kubeconfigs.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/merge-kubeconfigs.sh) to combine multiple `kubeconfigs` generated by `get-kubeconfig.sh`.
 
+<Admonition type="warning" title="Warning" >
```

Comment:
Let's remove this box from the "Generating kubeconfig" section since we moved it to the next section.